provider/provider_test.go

```diff
@@ -0,0 +1,46 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/pulumi/providertest"
+)
+
+func TestExamples(t *testing.T) {
+	t.Run("cluster", func(t *testing.T) {
+		test(t, "../examples/cluster").Run(t)
+	})
```

Comment:
I don't see any opportunity for the call to `test(...)` to fail. We should just be able to elide the `func` here:
```suggestion
	t.Run("cluster", test(t, "../examples/cluster").Run)
```

Reply:
Well well, you have to leave markers for the IDE to let you click on the test to run it. I really need to test it in VS Code.